backend/src/test/kotlin/com/infowings/catalog/data/aspect/AspectHistoryTest.kt

```diff
@@ -116,6 +120,9 @@ class AspectHistoryTest {
                 description = "some description-1"
             ), username
         )
+
+        Thread.sleep(10)
```

Comment:
What is the reason behind this `Thread.sleep(10)`?

Reply:
It is quick workaround for general problem of history elements ordering.
Now we use timestamping of this and it is not robust solution.
In particular it could happend that two events have the same timestamp.
In general we need some well founded solution. I guess it should be based on some kind of increasing events identifiers.

However, we have no resources for such kind of solution so I use quick workaround here.

I'll put comment to code.